front/src/form/common/components/custom-inputs/DateInput.tsx

```diff
@@ -23,13 +23,19 @@ export default function DateInput({
 } & ReactDatePickerProps) {
   const { value, ...rest } = field;
 
+  const setSelectedDate = () => {
```

Comment:
Est-ce que ce n'est pas plutot un `get` qu'un `set`?
```suggestion
  const getSelectedDate = () => {
```
Avec `set` j'ai cru que `selected={setSelectedDate()}` était une action et j'ai mis du temps à comprendre pourquoi ce n'était pas   `selected={() => setSelectedDate()}`. En fait c'est bien une valeur qui est attendue et la fonction fait un calcul de valeur. Il me semble que le mot clé `get` le refléterait mieux

Reply:
c'est fix